pages/about.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from '@testing-library/react';
+import About from './about.page';
+
+describe('<About />', () => {
+  it('should render without crashing', async () => {
+    render(<About />);
+
+    // Act
+    await screen.findByRole('heading');
```

Comment:
once you have more than 1 heading component on the screen, this will not work - you may wish to use the second parameter `{ name: 'About' }` to select the element, and then assert that it's in the DOM.

Also, since you're not doing anything with it and asserting against it, the act is actually the `render`, and the assert is everything after that:

`expect(await screen.findByRole('heading', { name: 'About' })).toBeInTheDocument()`

Reply:
Makes sense. I've adjusted this now. Thanks :)